src/main/java/hudson/plugins/warnings/WarningsDescriptor.java

```diff
@@ -150,8 +166,8 @@ public boolean isApplicable(final Class<? extends AbstractProject> jobType) {
      *
      * @return the Groovy parsers
      */
-    public GroovyParser[] getParsers() {
-        return groovyParsers.toArray(new GroovyParser[groovyParsers.size()]);
+    public List<GroovyParser> getParsers() {
```

Comment:
breaks binary compatibility

Reply:
Ditto on this, thank will look into it.

Reply:
you can have both and make GroovyParser[] version make a to call List<Groovy> with toArray() converter 👍